amundsen_application/static/js/ducks/allTags/api/v0.ts

```diff
@@ -1,17 +1,15 @@
-import axios, { AxiosResponse, AxiosError } from 'axios';
-
-import { AllTagsResponse } from '../types';
+import axios, { AxiosResponse } from 'axios';
 
 import { sortTagsAlphabetical } from 'ducks/utilMethods';
+import { TagsInterface } from 'interfaces/Tags';
+
+export type AllTagsResponseAPI = {
```

Comment:
I'm all in favor for keeping the API response types inside the API. I wonder if there's a way to enforce the import rules-- so none of the API types are imported except for the unit tests.

If we are using namespaces, we could create a namepsace just for testing and export that. Not sure about syntax but something like


```
type AllTagsResponseAPI = { ... } // private

export namespace TagsTestInterface {
  export AllTagsResponseAPI;
}
```
In this way the AllTagsResponseAPI can be imported in Tests, and anywhere else that tries to import this will have some janky looking code that is obviously very `import TagsTestInterface.AllTagsResponseAPI`.

Reply:
Open to any further investigation surrounding how we can clean up which interfaces are available globally. Yet this PR will not contain any of that work since it's a separate endeavor.